charts/seed-terraformer/charts/packet-infra/templates/_main.tf

```diff
@@ -0,0 +1,24 @@
+{{- define "packet-infra.main" -}}
+provider "packet" {
+  auth_token = "${var.PACKETAPIKEY}"
+}
+
+locals {
+  project_id = "{{ required "projectId is required" .Values.projectID }}"
+}
+
+// Import an existing public key
```

Comment:
"Import an existing"? This comment is incorrect, right? Aren't you creating a new SSH key here?

Reply:
Yes, quite. Bad comment. Fixed.